chartfx-chart/src/main/java/de/gsi/chart/plugins/EditAxis.java

```diff
@@ -256,15 +264,15 @@ public final void setZoomDuration(final Duration duration) {
             box.getChildren().add(getMinMaxButtons(axis, isHorizontal, false));
         }
 
-        private void changeAxisRange(final Axis axis, final boolean isIncrease) {
+        protected void changeAxisRange(final Axis axis, final boolean isIncrease) {
             final double width = Math.abs(axis.getMax() - axis.getMin());
 
             // TODO: check for linear and logarithmic axis
             changeAxisRangeLinearScale(width, axis.minProperty(), !isIncrease);
             changeAxisRangeLinearScale(width, axis.maxProperty(), isIncrease);
         }
 
-        private void changeAxisRangeLimit(final Axis axis, final boolean isHorizontal, final boolean isIncrease) {
+        protected void changeAxisRangeLimit(final Axis axis, final boolean isHorizontal, final boolean isIncrease) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'changeAxisRangeLimit(Axis, boolean, boolean)' has an NPath complexity of 576](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4989692)